src/UI/Art/SpriteArtist/SpriteLibrary.java

```diff
@@ -485,65 +483,48 @@ public static BufferedImage getHideIcon(Color color)
   //  Below is code for dealing with drawing in-game menus.
   ///////////////////////////////////////////////////////////////////
 
-  /**
-   * Returns a Sprite containing every uppercase letter, one letter per frame. The image
-   * is loaded, and the Sprite is created, on the first call to this function, and simply
-   * returned thereafter. Letters are stored in order starting from 'A'.
-   * @return A Sprite object containing the in-game menu font for uppercase letters.
-   */
-  public static Sprite getLettersUppercase()
-  {
-    if( null == letterSpritesUppercase )
-    {
-      letterSpritesUppercase = new Sprite(SpriteLibrary.loadSpriteSheetFile("res/ui/main/letters_uppercase.png"), 5, 11);
-    }
-    return letterSpritesUppercase;
-  }
-
-  /**
-   * Returns a Sprite containing every lowercase letter, one letter per frame. The image
-   * is loaded, and the Sprite is created, on the first call to this function, and simply
-   * returned thereafter. Letters are stored in order starting from 'a'. 
-   * @return A Sprite object containing the in-game menu font for lowercase letters.
-   */
-  public static Sprite getLettersLowercase()
+  public static PixelFont getFontStandard()
   {
-    if( null == letterSpritesLowercase )
+    if( null == fontStandard )
     {
-      letterSpritesLowercase = new Sprite(SpriteLibrary.loadSpriteSheetFile("res/ui/main/letters_lowercase.png"), 5, 11);
-    }
-    return letterSpritesLowercase;
-  }
-
-  /**
-   * Returns a Sprite containing the digits 0-9, one number per frame. The image
-   * is loaded, and the Sprite is created, on the first call to this function, and simply
-   * returned thereafter. Numbers are stored in order starting from 0.
-   * @return A Sprite object containing the in-game menu font for numbers.
-   */
-  public static Sprite getNumbers()
-  {
-    if( null == numberSprites )
-    {
-      numberSprites = new Sprite(SpriteLibrary.loadSpriteSheetFile("res/ui/main/numbers.png"), 5, 11);
+      try
+      {
+        final File folder = new File("res/");
+        if( folder.canRead() )
+        {
+          for( final File fileEntry : folder.listFiles() )
+          {
+            String filepath = fileEntry.getAbsolutePath();
+            // Look for files with our extension
+            if( !fileEntry.isDirectory() && filepath.endsWith(".ttf") )
+            {
+              // The name should be formatted as fontname_wxh.ttf
+              String[] nameparts = fileEntry.getName().split("[_x.]");
+
+              if( nameparts.length != 4 ) continue;  // Not formatted correctly.
```

Comment:
This does ban using underscores or dots in the rest of the file name. That's slightly awkward, but probably fine.

Reply:
Yup. We can either specify the font name format, or come up with a more contrived mechanism for specifying the render size. I'm pretty good with this approach.